docs/release_notes/5.0.15-13.md

```diff
@@ -0,0 +1,36 @@
+# Percona Server for MongoDB 5.0.15-13 (2023-03-14)
+
+| **Release date** | March 14, 2023 |
+|----------------- | ---------------- | 
+| **Installation** | [Installing Percona Server for MongoDB](../install/index.md)|
+
+Percona Server for MongoDB 5.0.15-13 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for [MongoDB 5.0.15 Community Edition](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.15---feb-27--2023).
+
+It supports MongoDB 5.0.14 protocols and drivers 
+
+## Release Highlights
+
+* [SERVER-54900](https://jira.mongodb.org/browse/SERVER-54900) - Fixed the issue with blocked networking calls preventing the oplog fetching process to re-establish the connection to an unresponsive sync-source node 
+* [SERVER-72416](https://jira.mongodb.org/browse/SERVER-72416) - Fixed the issue with incorrect projection parsing when a collection level collation is specified
+* [SERVER-71759](https://jira.mongodb.org/browse/SERVER-71759) - Changed the yielding policy of dataSize command to YIELD_AUTO for both when the command is called with estimate:true or false
+* [SERVER-72222](https://jira.mongodb.org/browse/SERVER-72222) - Fixed the incorrect behavior of the `mapReduce` command with single reduce optimization in sharded clusters
+
+Find the full list of changes in the [MongoDB 5.0.14 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.15---feb-27--2023).
+
+
+## New Features
+
+* [PSMDB-1033](https://jira.percona.com/browse/PSMDB-1033): Add authentication with AWS IAM
```

Comment:
@Jan should we mark it as tech preview?

Reply:
@sandraromanchenko I'm not sure, but maybe @janwieremjewicz can decide this.

Reply:
Sorry, my bad)